tests/sentry/models/test_user.py

```diff
@@ -11,12 +11,17 @@
 from sentry.testutils.silo import assume_test_silo_mode, control_silo_test
 
 
-@control_silo_test
+@control_silo_test(stable=True)
 class UserTest(TestCase):
     def test_hybrid_cloud_deletion(self):
         user = self.create_user()
+
+        # Determines the region to which the deletion cascades
+        organization = self.create_organization()
+        self.create_member(user=user, organization=organization)
+
         user_id = user.id
-        self.create_saved_search(name="some-search", owner=user)
+        self.create_saved_search(name="some-search", owner=user, organization=organization)
```

Comment:
By the way, linking the saved search to the organization here actually isn't necessary for the test to pass. I can leave it null, and the deletion will still work, I think because the test environment has only the one region. It's necessary only for the user to belong to an org on the correct region for the tombstone to be created, not necessarily for the organization actually to be linked to the tombstone's target. Kind of makes me want to flesh this out with more cases, and with multiple regions.